static/app/utils/useMetricsContext.tsx

```diff
@@ -0,0 +1,16 @@
+import {useContext} from 'react';
+
+import {MetricsContext} from 'sentry/utils/metrics/metricsContext';
+
+export function useMetricsContext() {
```

Comment:
How do you feel about naming this to `useMetrics()`? I dont think the consumer cares where the values are coming from here right, so context could be unnecessary